### PR TITLE
Doubleclick render idle throttle

### PR DESCRIPTION
**Pull request by keithwrightbos**
For doubleclick Fast Fetch experiment allowing render beyond 3 viewports when scheduler is idle.  Currently up to 4 amp-ad slots call layoutCallback simultaneously despite the possible render of multiple non-AMP validated creatives.  This PR addresses this inconsistency by enforcing same 1 second throttle if experiment is enabled (controlled via header on response).

### CODE REVIEW

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -409,6 +403,16 @@ export class AmpA4A extends AMP.BaseElement {
     return this.isRelayoutNeededFlag;
   }
 
+  /**
+   * @return {!Promise<boolean>} promise blocked on ad promise whose result is
+   *    whether creative returned is validated as AMP.
+   */
+  isVerifiedAmpCreativePromise() {
+    this.verifyStillCurrent();
```
Comment (jonkeller) [inline extensions/amp-a4a/0.1/amp-a4a.js]: This returns a function, but you're not saving the return value?

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ import {
   DEFAULT_SAFEFRAME_VERSION,
   assignAdUrlToError,
 } from '../../amp-a4a/0.1/amp-a4a';
+import {is3pThrottled} from '../../amp-ad/0.1/concurrent-load';
```
Comment (jonkeller) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Need exception in dep-check-config.js

Comment (jonkeller) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: ...actually, it looks like this dependency was specifically blacklisted, as opposed to just not whitelisted. What's up with that?
```
  {
    filesMatching: 'extensions/**/*-ad-network-*.js',
    mustNotDependOn: [
      'extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js',
      'extensions/amp-ad/0.1/concurrent-load.js',
      'src/3p-frame.js',
      'src/iframe-helper.js',
    ],
  },
```

Comment (keithwrightbos) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Fixed thanks for running it down (removed concurrent-load from list... not sure why its there)

**Pull request merged**